main.js

```diff
@@ -3,30 +3,56 @@
  */
 import kyOriginal from 'ky-universal';
 
+export const ky = kyOriginal;
+
 export const DEFAULT_HEADERS = {
   Accept: 'application/ld+json, application/json'
 };
 
-const ky = kyOriginal.create({headers: DEFAULT_HEADERS});
-
 const proxyMethods = new Set([
-  'get', 'post', 'push', 'patch', 'head', 'delete'
+  'get', 'post', 'put', 'push', 'patch', 'head', 'delete'
 ]);
 
-export const httpClient = new Proxy(ky, {
-  apply: _handleResponse,
-  get(target, propKey) {
-    const propValue = target[propKey];
+/**
+ * Returns a custom httpClient instance. Used to specify default headers and
+ * other default overrides.
+ *
+ * @param {object} [options={}] - Options hashmap.
+ * @param {object} [options.headers={}] - Default header overrides.
+ * @param {object} [options.httpsAgent] - Optional HTTPS agent.
+ * @param {object} [options.params] - Other default overrides.
+ *
+ * @return {httpClient} Custom httpClient instance.
+ */
+export function customClient({headers = {}, httpsAgent, ...params} = {}) {
+  // Ensure default headers, allow overrides
+  const ky = kyOriginal.create({
+    headers: {...DEFAULT_HEADERS, ...headers},
+    agent: httpsAgent,
+    ...params
+  });
+  return _createProxy({ky});
+}
+
+export const httpClient = _createProxy(
+  {ky: kyOriginal.create({headers: DEFAULT_HEADERS})});
```

Comment:
This could use `customClient`:
```suggestion
export const httpClient = customClient();
```